index.php

```diff
@@ -29,7 +29,16 @@
         <form action="index.php" method="post">
         <input class="cat-id-input" name="breedID" value="' . $cat->getBreed() . '">
         <input class="cat-id-input" name="newFavourite" value="' . $cat->getID() . '">
-        <img class="favorite-icon" src="images/fav-icon-empty.svg" alt="">
+        <img class="favorite-icon" src="';
+
+        $breedWanted = $_GET['breed'] - 1;
+
+        if($cat->getID() == $breeds[$breedWanted]['favourite_id']) {
+            $catshtml .= 'images/fav-icon-full.svg';
+            } else { $catshtml .= 'images/fav-icon-empty.svg';
+        }
+
+        $catshtml .=  '" alt="">
```

Comment:
alt not set

Reply:
this change was made in story 3 but story 3 is not merged correctly into story4